src/pages/@owner/account/AccountForm.tsx

```
@@ -147,44 +75,50 @@ export default function Account() {
           <Button
             variant='default'
             buttonStyle='edit'
-            children='초기화'
-            handleClickButton={handleClickReset}
+            className='px-[2rem]'
+            children='삭제'
+            handleClickButton={handleClickDelete}
```

Biome (2.1.2)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)